portal-web/docroot/html/taglib/ui/form_navigator/lexicon/sections.jspf

```diff
@@ -113,27 +113,26 @@
 				</c:otherwise>
 			</c:choose>
 
-			Liferay.once(
-				'<portlet:namespace />formReady',
-				function(event) {
-					if (!sectionContent.hasClass('in')) {
-						if (focusField.length) {
+			if (focusField.length) {
```

Comment:
I just pushed a new commit with the changes that you mentioned at 941660540938c9cc44f07ab5a3fd4482c3602db1.

I agree that we should open the panel regardless of the ```focusField`` being passed.